project-base/app/tests/App/Functional/Twig/PriceExtensionTest.php

```diff
@@ -154,17 +179,18 @@ public static function priceFilterMultidomainDataProvider(): array
     /**
      * @param \Shopsys\FrameworkBundle\Component\Money\Money $input
      * @param int $domainId
+     * @param string $locale
      * @param string $result
      */
     #[DataProvider('priceFilterMultidomainDataProvider')]
     #[Group('multidomain')]
-    public function testPriceFilterMultidomain(Money $input, int $domainId, string $result): void
+    public function testPriceFilterMultidomain(Money $input, int $domainId, string $locale, string $result): void
     {
         $this->domain->switchDomainById($domainId);
 
         $priceExtension = $this->getPriceExtensionWithMockedConfiguration();
 
-        $this->assertSame($result, $priceExtension->priceFilter($input));
+        $this->assertSame($result, $priceExtension->priceFilter($input, $locale));
```

Comment:
Is the `$locale` parameter added just for the purpose of the test? 
If so, can't we think of another way? I don't have a good feeling about changing class API for test only.

Reply:
I have reworked it